sns/ut/parity_math_ut.c

```diff
@@ -1433,49 +1433,111 @@ static void ub_large_32768(int iter)
 	fuc = duc+puc;
 	parity_math_tb();
 }
+static void ub_small_4_2_4K(int iter)
+{
+	UNIT_BUFF_SIZE = 4096;
+	duc = 4;
+	puc = 2;
+	fuc = duc+puc;
+	parity_math_tb();
+}
+
+static void ub_small_4_2_256K(int iter)
+{
+	UNIT_BUFF_SIZE = 262144;
```

Comment:
While the number 262144 represents 256K it would be nice if you instead write something like (256 * 1024) so that the number becomes more obvious for the reader. Similar observation for the next function where 1MB size is used.
I will still approve this code but if possible change this but do not ask for a re-review.

Reply:
Sure @shashank-parulekar. I will address this in [EOS-18143](https://jts.seagate.com/browse/EOS-18143).